src/service/company.managing.officer.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Session } from "@companieshouse/node-session-handler";
+import { Request } from "express";
+import { makeApiCallWithRetry } from "./retry.handler.service";
+import { CompanyOfficersResource } from "@companieshouse/api-sdk-node/dist/services/company-officers/types";
+import { createAndLogErrorRequest, logger } from "../utils/logger";
+
+export const getCompanyOfficers = async (req: Request, companyNumber: string): Promise<CompanyOfficersResource | undefined> => {
+  logger.debugRequest(req, `Retrieving list of officers for company: ${companyNumber}`);
+
+  const maxResults = 50;
```

Comment:
Why 50? Can we pull this out into a constant (if not configurable then easily identified / commented). I'd say 100 would be adequate.

Reply:
Why 100? Chats with Matthew indicated that although no upper limit defined, we could expect "low double figures".